plutus-core-spec/main.tex

```diff
@@ -875,6 +888,18 @@ \section{Syntax}
 
 \subfile{figures/PlutusCoreGrammar}
 
+%%%%%%%
+\red{
+Some comments on the syntax...
+\begin{itemize}
+\item Are Values also Terms?  If not, then I don't think $\wrap{\alpha}{S}{V}$ is a term, for example (perhaps this is correct).
```

Comment:
They certainly are in the implementation and I think the same holds for the specification.